doc/user-guide/io.rst

```diff
@@ -741,6 +741,64 @@ instance and pass this, as follows:
 .. _Google Cloud Storage: https://cloud.google.com/storage/
 .. _gcsfs: https://github.com/fsspec/gcsfs
 
+.. _io.zarr.distributed_writes:
+
+Distributed writes
```

Comment:
With this title; I think this section should mention that `to_zarr` with a dask-backed dataset will do a parallel and/or distributed write depending on how your dask cluster is set up.

Reply:
Yes great idea